### PR TITLE
Fix to follow the brace-style ESLint rule

### PR DESCRIPTION
**Pull request by ruinername**
Fixes to follow the brace-style rule (https://eslint.org/docs/rules/brace-style)

### CODE REVIEW

File: root/static/scripts/relationship-editor/common/dialog.js [hunk 1]
```diff
@@ -263,8 +263,16 @@ const RE = MB.relationshipEditor = MB.relationshipEditor || {};
                             if (gid === target.gid) {
                                 _.each(entity.displayableRelationships(vm)(), function (r) {
                                     switch (relationshipFilter) {
-                                      case 'same-entity-types': if (r.entityTypes !== relationship.entityTypes) { return; }; break;
-                                      case 'same-relationship-type': if (r.linkTypeID() !== relationship.linkTypeID()) { return; }; break;
+                                      case 'same-entity-types':
+                                          if (r.entityTypes !== relationship.entityTypes) {
+                                              return;
+                                          }
+                                      break;
```
Comment (reosarevok) [inline root/static/scripts/relationship-editor/common/dialog.js]: I guess this break (and the one under it) should be indented with the if block, not the case? :)

File: root/static/scripts/relationship-editor/common/multiselect.js [hunk 2]
```diff
@@ -95,7 +95,9 @@ import deferFocus from '../../edit/utility/deferFocus';
 
                 var visible = matchIndex(option, term) >= 0 && (
                     linkedEntities.link_attribute_type[typeGID].creditable ||
-                    _.findIndex(selected, function (a) { return a.type.gid === typeGID }) < 0
+                    _.findIndex(selected, function (a) {
+                        return a.type.gid === typeGID;
+                    }) < 0
```
Comment (reosarevok) [inline root/static/scripts/relationship-editor/common/multiselect.js]: I'd personally prefer to move both selected and function into their own lines here, if you're ok with it? 

```
                    _.findIndex(
                        selected,
                        function (a) {
                            return a.type.gid === typeGID;
                        },
                    ) < 0
```

I feel it makes the comparison and whatnot clearer.

I might also move the closing ); to be just after the `< 0`, if eslint is happy with that.

File: root/static/scripts/relationship-editor/generic.js [hunk 3]
```diff
@@ -93,20 +93,32 @@ const RE = MB.relationshipEditor = MB.relationshipEditor || {};
         event: function (relationships, series) {
             return _.sortBy(
                 relationships,
-                function (r) { return r.target(series).begin_date || '' },
-                function (r) { return r.target(series).end_date || '' },
-                function (r) { return r.target(series).time || '' }
+                function (r) {
+                    return r.target(series).begin_date || '';
+                    },
```
Comment (reosarevok) [inline root/static/scripts/relationship-editor/generic.js]: This bracket (and the one under it) seem misindented.

File: root/static/scripts/relationship-editor/generic.js [hunk 4]
```diff
@@ -93,20 +93,32 @@ const RE = MB.relationshipEditor = MB.relationshipEditor || {};
...
+                function (r) {
+                    return r.target(series).end_date || '';
+                    },
+                function (r) {
+                    return r.target(series).time || '';
+                }
```
Comment (reosarevok) [inline root/static/scripts/relationship-editor/generic.js]: This should probably also have a comma (for consistent comma-dangle).

File: root/static/scripts/relationship-editor/generic.js [hunk 5]
```diff
@@ -93,20 +93,32 @@ const RE = MB.relationshipEditor = MB.relationshipEditor || {};
...
             );
         },
         release: function (relationships, series) {
             return _.sortBy(
                 relationships,
-                function (r) { return _(r.target(series).events).map(getDate).sort().head() },
-                function (r) { return _(r.target(series).labels).map(getCatalogNumber).sort().head() }
+                function (r) {
+                    return _(r.target(series).events).map(getDate).sort().head();
+                    },
```
Comment (reosarevok) [inline root/static/scripts/relationship-editor/generic.js]: Indent

File: root/static/scripts/relationship-editor/release.js [hunk 6]
```diff
@@ -206,7 +203,11 @@ const RE = MB.relationshipEditor = MB.relationshipEditor || {};
             }));
 
             var trackCount = _.reduce(release.mediums(),
-                function (memo, medium) { return memo + medium.tracks.length }, 0);
+                function (memo, medium) {
+                    return memo + medium.tracks.length;
+                },
+                0
```
Comment (reosarevok) [inline root/static/scripts/relationship-editor/release.js]: I'd also move release.mediums() to its own line, and you'd want a comma-dangle after 0 while you're at it.

File: root/static/scripts/release-editor/edits.js [hunk 7]
```diff
@@ -422,15 +421,19 @@ releaseEditor.getEditPreviews = function () {
         releaseEditor.editPreviews(_.compact(_.map(edits, getPreview)));
     }
 
-    function getPreview(edit) { return previews[edit.hash] }
+    function getPreview(edit) {
+        return previews[edit.hash];
+    }
```
Comment (reosarevok) [inline root/static/scripts/release-editor/edits.js]: The previous preview is separated with a newline, so let's try and be consistent here too :) Same below (we could also remove all newlines, but I at least feel newlines make it more clear)

File: root/static/scripts/tests/relationship-editor.js [hunk 8]
```diff
@@ -138,8 +142,10 @@ function setupGenericRelationshipEditor(options) {
 
 function formData() {
     var inputsArray = _.toArray($("input[type=hidden]"));
-    return _.transform(inputsArray, function (result, input) { result[input.name] = input.value }, {});
-};
+    return _.transform(inputsArray, function (result, input) {
+        result[input.name] = input.value;
+    }, {});
```
Comment (reosarevok) [inline root/static/scripts/tests/relationship-editor.js]: I'd move each argument to its own line(s) here.

File: root/static/scripts/edit/MB/edit.js [hunk 9]
```diff
@@ -140,7 +150,9 @@ import request from '../../common/utility/request';
 
             data.attributes = _(ko.unwrap(relationship.attributes))
                 .invokeMap('toJS')
-                .sortBy(function (a) { return a.type.id })
+                .sortBy(function (a) {
+                    return a.type.id;
+                })
```
Comment (mwiencek) [inline root/static/scripts/edit/MB/edit.js]: I think the inline function was fine (perhaps we should enable `allowSingleLine` on the rule), but we could also just replace it with an arrow function for simplicity:

```JS
.sortBy(a => a.type.id)
```

So I'd go with that.

File: root/static/scripts/edit/MB/edit.js [hunk 10]
```diff
@@ -329,7 +341,9 @@ import request from '../../common/utility/request';
     edit.releaseAddReleaseLabel = editConstructor(
         TYPES.EDIT_RELEASE_ADDRELEASELABEL,
 
-        function (args) { delete args.release_label }
+        function (args) {
+            delete args.release_label;
+        }
```
Comment (mwiencek) [inline root/static/scripts/edit/MB/edit.js]: Mind adding a comma here (`},`) to satisfy the comma-dangle rule?

File: root/static/scripts/edit/MB/edit.js [hunk 11]
```diff
@@ -382,7 +396,9 @@ import request from '../../common/utility/request';
 
     edit.relationshipCreate = editConstructor(
         TYPES.EDIT_RELATIONSHIP_CREATE,
-        function (args) { delete args.id }
+        function (args) {
+            delete args.id;
+        }
```
Comment (mwiencek) [inline root/static/scripts/edit/MB/edit.js]:
```suggestion
        },
```

File: root/static/scripts/edit/utility/guessFeat.js [hunk 12]
```diff
@@ -50,7 +50,9 @@ function extractNonBracketedFeatCredits(str, artists, isProbablyClassical) {
 
     var artistCredit = wrapped
         .splice(2)
-        .filter(function (value, key) { return key %2 == 0; })
+        .filter(function (value, key) {
+            return key %2 == 0;
+        })
         .compact()
```
Comment (mwiencek) [inline root/static/scripts/edit/utility/guessFeat.js]: `.filter((value, key) => key % 2 === 0)` would be nicer here

File: root/static/scripts/guess-case/MB/Control/GuessCase.js [hunk 13]
```diff
@@ -34,9 +34,13 @@ MB.Control.initialize_guess_case = function (type, formPrefix) {
     }
 
     $name.parent()
-        .find("button.guesscase-title").on("click", function () { setVal($name, guess.guess($name.val())) })
+        .find("button.guesscase-title").on("click", function () {
```
Comment (mwiencek) [inline root/static/scripts/guess-case/MB/Control/GuessCase.js]: Mind starting `.on` on the next line (at the same indentation level as the other method calls) so it doesn't conflict with #1311? :)

File: root/static/scripts/guess-case/MB/Control/GuessCase.js [hunk 14]
```diff
@@ -34,9 +34,13 @@ MB.Control.initialize_guess_case = function (type, formPrefix) {
...
+            setVal($name, guess.guess($name.val()));
+        })
         .end()
-        .find("button.guesscase-options").on("click", function () { $options.dialog("open") });
+        .find("button.guesscase-options").on("click", function () {
```
Comment (mwiencek) [inline root/static/scripts/guess-case/MB/Control/GuessCase.js]: Same here re: `.on`

File: root/static/scripts/guess-case/MB/GuessCase/Handler/Base.js [hunk 15]
```diff
@@ -883,7 +883,9 @@ MB.GuessCase.Handler.Base = function (gc) {
     self.moveArticleToEnd = function (is) {
         return utils.trim(is).replace(
             /^(The|Los) (.+)$/,
-            function (match, article, name) { return name + ", " + article }
+            function (match, article, name) {
+                return name + ", " + article;
+            }
```
Comment (mwiencek) [inline root/static/scripts/guess-case/MB/GuessCase/Handler/Base.js]:
```suggestion
            },
```

File: root/static/scripts/timeline.js [hunk 16]
```diff
@@ -583,7 +620,11 @@ class TimelineLine {
                 } else if (graph === 'overview') {
                     options.series = { lines: { lineWidth: 1 }, shadowSize: 0 };
                     options.xaxis = { mode: "time", minTickSize: [1, "year"] };
-                    options.yaxis = { tickFormatter: function () { return '' } };
+                    options.yaxis = {
+                        tickFormatter: function () {
+                            return '';
+                        }
+                    };
```
Comment (mwiencek) [inline root/static/scripts/timeline.js]: This was changed to `{ tickFormatter: () => '' };` in the master branch via another PR, so can be reverted.

File: root/static/scripts/edit/MB/CoverArt.js [hunk 17]
```diff
@@ -19,41 +19,30 @@ MB.CoverArt.get_image_mime_type = function () {
...
          * data in the index is incorrect.
          */
         $img.attr("src", require('../../../images/image404-125.png'));
-    }
-    else
-    {
-        $img.attr("src", image.image);
-    }
+    } else {
+        $img.attr("src", image.image)
```
Comment (mwiencek) [inline root/static/scripts/edit/MB/CoverArt.js]: The closing `}` here got removed

File: root/static/scripts/relationship-editor/common/multiselect.js [hunk 18]
```diff
@@ -95,8 +95,12 @@ import deferFocus from '../../edit/utility/deferFocus';
...
                 var visible = matchIndex(option, term) >= 0 && (
                     linkedEntities.link_attribute_type[typeGID].creditable ||
-                    _.findIndex(selected, function (a) { return a.type.gid === typeGID }) < 0
-                );
+                    _.findIndex(
+                        selected,
+                        function (a) {
+                            return a.type.gid === typeGID;
+                        },
+                    ) < 0);
```
Comment (mwiencek) [inline root/static/scripts/relationship-editor/common/multiselect.js]: I think keeping it on one line and using an arrow function would be much more readable here
```JS
_.findIndex(selected, a => a.type.gid === typeGID) < 0
```

File: root/static/scripts/edit/MB/CoverArt.js [hunk 19]
```diff
@@ -167,24 +155,21 @@ MB.CoverArt.validate_file = function (file) {
...
-        if ((uint32view[0] & 0x00FFFFFF) === 0x00FFD8FF)
-        {
+        if ((uint32view[0] & 0x00FFFFFF) === 0x00FFD8FF) {
             deferred.resolve('image/jpeg');
-        }
-        else if (uint32view[0] === 0x38464947) /* GIF signature. "GIF8" */
-        {
+        } else if (uint32view[0] === 0x38464947) {
+            // GIF signature. "GIF8"
+
```
Comment (mwiencek) [inline root/static/scripts/edit/MB/CoverArt.js]: Can we also removed the added newlines here?

File: root/static/scripts/relationship-editor/common/entity.js [hunk 20]
```diff
@@ -52,13 +52,17 @@ const RE = MB.relationshipEditor = MB.relationshipEditor || {};
...
+                })
                 .compact()
                 .value();
 
             var allRelationships = _(this.relationships.peek())
                 .union(newRelationships)
-                .sortBy(function (r) { return r.lowerCasePhrase(self) })
+                .sortBy(function (r) {
+                  return r.lowerCasePhrase(self);
+                })
```
Comment (mwiencek) [inline root/static/scripts/relationship-editor/common/entity.js]:
```JS
.sortBy(r => r.lowerCasePhrase(self))
```

File: root/static/scripts/relationship-editor/common/entity.js [hunk 21]
```diff
@@ -52,13 +52,17 @@ const RE = MB.relationshipEditor = MB.relationshipEditor || {};
             }
 
             var newRelationships = _(relationships)
-                .map(function (data) { return MB.getRelationship(data, self) })
+                .map(function (data) {
+                  return MB.getRelationship(data, self);
+                })
```
Comment (mwiencek) [inline root/static/scripts/relationship-editor/common/entity.js]:
```JS
.map(data => MB.getRelationship(data, self))
```

File: root/static/scripts/relationship-editor/common/viewModel.js [hunk 22]
```diff
@@ -127,7 +127,9 @@ export class ViewModel {
 
         _sortedRelationships(relationships, source) {
             return relationships
-                .sortBy(function (r) { return r.lowerCaseTargetName(source) })
+                .sortBy(function (r) {
+                    return r.lowerCaseTargetName(source);
+                })
```
Comment (mwiencek) [inline root/static/scripts/relationship-editor/common/viewModel.js]:
```JS
.sortBy(r => r.lowerCaseTargetName(source))
```

**Comment by mwiencek** [comment]
I suggested some cases where keeping things on one line but using an arrow function would improve readability.

**Comment by mwiencek** [comment]
This has been merged in https://github.com/metabrainz/musicbrainz-server/commit/87f0ab51284542670ce8ffe139b2e5b7297008a4. Thank you! :)

**Pull request closed without merging**